### PR TITLE
WIP: InfoPros docs rewrite

### PR DESCRIPTION
**Pull request by electronwill**
WIP: InfoPros docs rewrite

### CODE REVIEW

**Comment by electronwill** [comment]
Work in progress. Please do not merge.

Note: Need to rename all images to have a conda- prefix, and all references to those images in the files.
Note: Need to check any changes to .py files. conf.py, scrape_help.py.
Note: Check deletion of conda-pip-virtualenv-translator.html.

Otherwise, follow standard InfoPros rewrite procedure-- complete review, merging in changes made since the InfoPros forked the docs, redirects, and so on.

**Comment by kerrywatson1** [comment]
@electronwill turning this over to you for final resolution and launch!

Of the 44 items in history, I reconciled and updated 26. The remaining 18 items in history are largely new sections on conda-build, which you have been involved with and can edit better than I can. 

I'll review with you on Monday.

**Comment by electronwill** [comment]
OK, sounds good. I think the new conda-build sections are separate enough that I can add them without needing to make significant changes to the rest of the docs in the PR. Will talk more with you about it Monday.

File: docs/source/admin-guide/configuration/use-condarc.rst [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+-----------------------------------
+
+Normally the default repository is `repo.continuum.io
+<http:repo.continuum.io>`_, but if default_channels is defined,
+it sets the new list of default channels. This is especially
+useful for air gap and enterprise installations:
+
+.. code-block:: yaml
+
+  channels:
```
Comment (183amir) [inline docs/source/admin-guide/configuration/use-condarc.rst]: Shouldn't `channels` here be `default_channels`?

**Comment by electronwill** [comment]
- Turns out the "specified below" was an error from way back in https://github.com/conda/conda-docs/pull/87 from May 2015, when the notes section was moved from the top of the file to the bottom, and the "specified below" was not changed to "specified above".
- All commits from https://github.com/conda/conda-docs/commits/master/docs/source/config.rst are included in use-condarc.rst.

**Comment by electronwill** [comment]
The last bit is worrying. The source file at https://github.com/conda/conda-docs/blob/master/docs/source/spec.rst said Unix. The infopros seem to have changed this to Linux, which is incorrect since both Linux and macOS are types of Unix. On one instance they tagged us with the misspelled "@cios-docs" asking if macOS should be included as well, and on the other instance they didn't tag us at all. I'll continue looking for this sort of thing as I skim these files.

**Comment by electronwill** [comment]
There are 61 changed files that we could create redirects for: https://gist.github.com/electronwill/09efbd6a7405c1090a71bdae20b6cbe6

When I mentioned redirects to Kale, he said that currently conda.io is not on readthedocs although there is a plan to move it there in the future. So we don't get the easy RTD interface for redirects, but he said there is a way to create them using the system on the current server.

I could also create 61 files that each contain an html redirect, but it would make the conda-docs repo pretty cluttered and make it tough for contributors to find where to make their changes, so I'd rather use the system on the current server.

I vote we discuss this with him when we meet for the review.

**Comment by electronwill** [comment]
CI passes and no merge conflicts. This is now ready for the meeting Monday.

**Comment by electronwill** [comment]
redirects

https://conda.io/docs/admin.html
https://conda.io/docs/user-guide/configuration/admin-multi-user-install.html

https://conda.io/docs/announcements.html
https://conda.io/docs/help-support.html

https://conda.io/docs/bdist_conda.html
https://conda.io/docs/user-guide/tasks/build-packages/build-without-recipe.html

https://conda.io/docs/build-commands.html
https://conda.io/docs/commands.html

https://conda.io/docs/build.html
https://conda.io/docs/user-guide/tasks/build-packages/recipe.html

https://conda.io/docs/build_tutorials.html
https://conda.io/docs/user-guide/tutorials/index.html

https://conda.io/docs/changelog.html
https://conda.io/docs/release-notes.html

https://conda.io/docs/channels.html
https://conda.io/docs/user-guide/tasks/manage-channels.html

https://conda.io/docs/config.html
https://conda.io/docs/user-guide/configuration/index.html

https://conda.io/docs/custom-channels.html
https://conda.io/docs/user-guide/tasks/create-custom-channels.html

https://conda.io/docs/download.html
https://conda.io/docs/user-guide/install/download.html

https://conda.io/docs/env-commands.html
https://conda.io/docs/commands.html

https://conda.io/docs/faq.html
https://conda.io/docs/user-guide/tasks/index.html

https://conda.io/docs/general-commands.html
https://conda.io/docs/commands.html

https://conda.io/docs/get-involved.html
https://conda.io/docs/help-support.html

https://conda.io/docs/get-started.html
https://conda.io/docs/user-guide/index.html

https://conda.io/docs/installation.html
https://conda.io/docs/user-guide/install/index.html

https://conda.io/docs/intro.html
https://conda.io/docs/index.html

https://conda.io/docs/mro.html
https://conda.io/docs/user-guide/tasks/use-mro-with-conda.html

https://conda.io/docs/py2or3.html
https://conda.io/docs/user-guide/tasks/manage-python.html

https://conda.io/docs/r-with-conda.html
https://conda.io/docs/user-guide/tasks/use-r-with-conda.html

https://conda.io/docs/signed-packages.html
https://conda.io/docs/user-guide/tasks/build-packages/sign-packages.html

https://conda.io/docs/spec.html
https://conda.io/docs/user-guide/tasks/build-packages/package-spec.html

https://conda.io/docs/test-drive.html
https://conda.io/docs/user-guide/getting-started.html

https://conda.io/docs/travis.html
https://conda.io/docs/user-guide/tasks/use-conda-with-travis-ci.html

https://conda.io/docs/troubleshooting.html
https://conda.io/docs/user-guide/troubleshooting.html

https://conda.io/docs/winxp-proxy.html
https://conda.io/docs/user-guide/configuration/use-winxp-with-proxy.html

https://conda.io/docs/build_tutorials/app.html
https://conda.io/docs/user-guide/tutorials/build-apps.html

https://conda.io/docs/build_tutorials/build_nltk.html
https://conda.io/docs/user-guide/tutorials/build-nltk.html

https://conda.io/docs/build_tutorials/pkgs.html
https://conda.io/docs/user-guide/tutorials/build-pkgs-skeleton.html

https://conda.io/docs/build_tutorials/pkgs2.html
https://conda.io/docs/user-guide/tutorials/build-pkgs.html

https://conda.io/docs/build_tutorials/postgis.html
https://conda.io/docs/user-guide/tutorials/build-postgis.html

https://conda.io/docs/build_tutorials/windows.html
https://conda.io/docs/user-guide/tutorials/build-windows.html

https://conda.io/docs/building/bpp.html
https://conda.io/docs/user-guide/tasks/build-packages/index.html

https://conda.io/docs/building/build-scripts.html
https://conda.io/docs/user-guide/tasks/build-packages/add-scripts.html

https://conda.io/docs/building/build.html
https://conda.io/docs/user-guide/tasks/build-packages/index.html

https://conda.io/docs/building/dirs.html
https://conda.io/docs/user-guide/concepts.html

https://conda.io/docs/building/environment-vars.html
https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html

https://conda.io/docs/building/meta-pkg.html
https://conda.io/docs/glossary.html

https://conda.io/docs/building/meta-yaml.html
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html

https://conda.io/docs/building/pkg-name-conv.html
https://conda.io/docs/user-guide/tasks/build-packages/package-naming-conv.html

https://conda.io/docs/building/recipe.html
https://conda.io/docs/user-guide/tasks/build-packages/recipe.html

https://conda.io/docs/building/sample-recipes.html
https://conda.io/docs/user-guide/tasks/build-packages/sample-recipes.html

https://conda.io/docs/building/shared-libraries.html
https://conda.io/docs/user-guide/tasks/build-packages/use-shared-libraries.html

https://conda.io/docs/building/variants.html
https://conda.io/docs/user-guide/tasks/build-packages/variants.html

https://conda.io/docs/help/conda-pip-virtualenv-translator.html
https://conda.io/docs/commands.html

https://conda.io/docs/help/help.html
https://conda.io/docs/help-support.html

https://conda.io/docs/help/silent.html
https://conda.io/docs/user-guide/install/index.html

https://conda.io/docs/help/support.html
https://conda.io/docs/help-support.html

https://conda.io/docs/install/central.html
https://conda.io/docs/user-guide/configuration/admin-multi-user-install.html

https://conda.io/docs/install/full.html
https://conda.io/docs/user-guide/install/index.html

https://conda.io/docs/install/quick.html
https://conda.io/docs/user-guide/install/index.html

https://conda.io/docs/install/sample-condarc.html
https://conda.io/docs/user-guide/configuration/sample-condarc.html

https://conda.io/docs/install/tab-completion.html
https://conda.io/docs/user-guide/configuration/enable-tab-completion.html

https://conda.io/docs/using/cheatsheet.html
https://conda.io/docs/user-guide/cheatsheet.html

https://conda.io/docs/using/envs.html
https://conda.io/docs/user-guide/tasks/manage-environments.html

https://conda.io/docs/using/index.html
https://conda.io/docs/user-guide/tasks/index.html

https://conda.io/docs/using/pkgs.html
https://conda.io/docs/user-guide/tasks/manage-pkgs.html

https://conda.io/docs/using/test-drive.html
https://conda.io/docs/user-guide/getting-started.html

https://conda.io/docs/using/using.html
https://conda.io/docs/user-guide/tasks/manage-conda.html

**Comment by kalefranz** [comment]
Woot! 🎉

**Pull request merged**